Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:16.04
```

Comment:
not really necessary for functioning, but I'd like to have it here for easier answers to issues of the sort "roboschool installation from source does not work on ubuntu 16.04!"